.github/workflows/repos.yml

```diff
@@ -0,0 +1,64 @@
+name: Preview or update Pulumi app nf-core/github-repos/prod
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "pulumi/github/repos/**/*"
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "pulumi/github/repos/**/*"
```

Comment:
Just to understand. This will run on PRs to `ops`, right? Is this action updating the pipeline health? And are these PRs on `ops` related to pipeline health?